build.gradle.kts

```diff
@@ -508,7 +508,12 @@ tasks {
     description = "Runs the integration tests."
     sharedIntegrationTestConfig(buildCodyDir, "replay")
     dependsOn("processIntegrationTestResources")
-    project.properties["repeatTests"]?.let { systemProperty("repeatTests", it) }
+    project.properties["repeatTests"]?.let {
+      systemProperty("repeatTests", it)
+      // We expect some of the repetitions to fail,
```

Comment:
Is there any practical benefit of that change?
I would say we *temporarily* expect them to fail, but in general we want all them to succeed no matter how many times we rerun it.

Reply:
I expect this temporary expectation to stay with us for a long time (but if I'm wrong, we can easily revert this change).

In the meantime, the practical benefit looks like this. Can you tell which runs have completed normally, and which ones have crashed? (There's more than one of each type!)
<img width="1080" alt="Screenshot 2024-07-17 at 16 25 12" src="https://github.com/user-attachments/assets/4dd0cb61-70ba-4d49-a23c-e8ea12308ace">